### ISSUE TITLE
Hotkey login

### ISSUE THREAD

**Issue by RobKuebler** [sarahavilov/logmein#2]
Hi!

I have found this already and downloaded the version you linked there:
https://addons.mozilla.org/en-us/firefox/addon/log-me-in/reviews/744854/

But still it doesn't work. I put in alt-n and also 'alt-n', but nothing happens on the login page. The key icon works though.

Firefox 41
Fedora 22

**Comment by sarahavilov** [comment]
Use a more complex hotkey. Probably the `alt-n` has already been used by Firefox or one of its extensions.

**Comment by RobKuebler** [comment]
Hi!

First of all, thanks for the very fast response, I can use this great extension now. :) No more 10+ password prompts. 

Now to this problem: I have chosen alt+n, because that was also the hotkey in Secure Login, which I ditched now for this one. ;) But I just tested other combinations, but it doesn't work for me. Also not stuff like control-alt-y or any of the example combinations on the addon settings page, also not after restarting after changing this setting. The key icon works like a charm though.

**Comment by sarahavilov** [comment]
Is the login window in a popup? or just a normal browser tab?

**Comment by RobKuebler** [comment]
Normal browser tab, for example here (just some German math website ;)): http://matheraum.de/login
But also here on Github for example.

**Comment by sarahavilov** [comment]
I just checked the http://matheraum.de/login website and it works fine for me on Mac and Windows. I even used your hotkey combinations, `control-alt-y`.

I made a debug version for you. Download and install it, then open the error console (Ctrl + Shift + J). Whenever you press the hotkey combinations, a new message is supposed to be shown in the console, `ilogmein:log-me-in is called from Hotkey.`

﻿https://www.dropbox.com/s/dgxzaej42fy7ll3/extension.xpi?dl=0

**Comment by RobKuebler** [comment]
Ok that's really weird, not it works... I tried again before the debug version, with different combinations and even restarting the browser, but it just didn't want to work. Then I installed the debug version, but couldn't see any messages in the error console, caused by your extension.

Then I restarted Firefox again and then it started working (also with alt-n, which I originally wanted). Then I removed the debugging version again and used the other one again (where the multiple Master password windows were fixed). Now it also works in this version! :) Maybe I had some weird settings, because I had the version from the mozilla website first... 

Anyway, thank you very very much, it works like a charm now! Great addon, great support! Thanks for your time :+1:
